### ISSUE TITLE
not in a resolved environment on Windows

### ISSUE THREAD

**Issue by brumenn** [AcademySoftwareFoundation/rez#452]
Hi !

Maybe I'm missing something, but after installing rez and do a quick `rez-bind --quickstart`, when I set an environment with `rez-env`, python for example, aliases work great but it's seems like no environment is keeped.

```
C:\Users\Sam>rez env python
resolved by Sam@MSI, on Sun Aug 13 16:17:11 2017, using Rez v2.13.1

requested packages:
python
~platform==windows       (implicit)
~arch==AMD64             (implicit)
~os==windows-10.0.14393  (implicit)

resolved packages:
arch-AMD64             C:\Users\Sam\packages\arch\AMD64                                                       (local)
os-windows-10.0.14393  C:\Users\Sam\packages\os\windows-10.0.14393                                            (local)
platform-windows       C:\Users\Sam\packages\platform\windows                                                 (local)
python-2.7.13          C:\Users\Sam\packages\python\2.7.13\platform-windows\arch-AMD64\os-windows-10.0.14393  (local)

C:\Users\Sam>rez context
not in a resolved environment context.

C:\Users\Sam>python
'python' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
```

So, dependency packages don't work as well, when I call one that need other packages, previous env are not keeped ..
Sorry maybe it's look like a noob issue ?

Thanks,
Nicolas

**Comment by NicolasCastelli** [comment]
Ok, because I use rez in Linux normally, I was a bit confused by that, but it seems to work great with `-c CMD` as in #375 . Some fixes added : #377 .

**Comment by nerdvegas** [comment]
Ok yeah looks as though #377 adds the necessary fixes for this, although I
think as well that we should drop sh/pbs completely, they're just fancy
subproc handlers and it doesn't seem worth the trouble they're causing on
Windows.

Should be able to start addressing these issues in the coming weeks, away
on vacation atm.

Cheers
A

**Comment by instinct-vfx** [comment]
I did some work on that already for our pre-production deployment branch. The main open question is about the classmethod for export(). On vacation too though, so greetings from Malta and have a good one Alan :)

**Comment by instinct-vfx** [comment]
I should have been clearer. By "work on that" i meant dropping sh.

**Comment by instinct-vfx** [comment]
For reference i created a separate issue for dropping sh as i ran into things i am not sure i understand heh #459

**Comment by skral** [comment]
Just confirmed that the example @NicolasCastelli gave has been fixed with #377 and the issue is solved when using version 2.18.0.

I think we can close this issue.

**Comment by JeanChristopheMorinPerso** [comment]
Fixed by https://github.com/AcademySoftwareFoundation/rez/pull/377
